tests/test_decap.py

```diff
@@ -0,0 +1,116 @@
+import pytest
+from netaddr import *
+import time
+import logging
+import requests
+from ptf_runner import ptf_runner
+from ansible_host import AnsibleHost
```

Comment:
There are already better replacements `duthost`, `ptfhost` and `testbed_devices` for the `ansible_host` module. Suggest to stop using `ansible_host`.